helm/hpcc/Chart.yaml

```diff
@@ -10,4 +10,23 @@ version: 0.1.0
 
 # This is the version number of the application being deployed. This version number should be
 # incremented each time you make changes to the application.
+
 appVersion: 0.1.0
+
+# Requires 'helm dependency update' command
+dependencies:
+# An Elastic Stack environment is managed by default
+# To disable: --set global.logging.managedElasticStack.enabled=false --set global.logging.managedElasticStack.kibana.enabled=false
+# and speficy alternate log processor
+- name: filebeat # Managed stderr log-forwarder
+  version: 7.9.3
+  repository: https://helm.elastic.co
+  condition: global.logging.managedElasticStack.enabled
+- name: elasticsearch # Managed logging processor engine
+  version: 7.9.3
+  repository: https://helm.elastic.co
+  condition: global.logging.managedElasticStack.enabled
```

Comment:
It seems the recommended convention here is to use elasticstack.enabled rather than global.logging.managedElasticStack.enabled. Filebeat can use the same condition, kibana should use kibana.enabled,elasticstack.enabled to check both.